### PR TITLE
operator multi-nic-cni-operator (1.2.3)

### PR DESCRIPTION
**Pull request by sunya-ch**
update compatible OCP version. 

---
Thanks for submitting your Operator. Please check the below list before you create your Pull Request.

### Your submission should not

* [x] Add more than one operator bundle per PR
* [x] Modify any operator
* [x] Rename an operator
* [x] Modify any files outside the above-mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description of the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human-readable name and 1-liner description about your Operator
* [x] Valid [category](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories) name <sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by sunya-ch** [comment]
Does anyone could help on updating compatible OCP version for existing bundle?

File: operators/multi-nic-cni-operator/1.2.3/metadata/annotations.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify OCP versions compatibility
-  com.redhat.openshift.versions: v4.12-v4.14
\ No newline at end of file
+  com.redhat.openshift.versions: v4.12-v4.18
```
Comment (bthurber) [inline operators/multi-nic-cni-operator/1.2.3/metadata/annotations.yaml:17]: com.redhat.openshift.versions: " >=v4.12, <=v4.18"

**Review by bthurber — CHANGES_REQUESTED** [review]
Update the annotations.yaml with the provided syntax and rerun the pipeline.

**Comment by Allda** [comment]
@sunya-ch Update of existing bundle is not supported action. Once an operator bundle is release you can't change it.

If you want to update existing operators you would need to switch to File Based Catalog - https://github.com/redhat-openshift-ecosystem/community-operators-prod/discussions/5031

**Comment by sunya-ch** [comment]
> @sunya-ch Update of existing bundle is not supported action. Once an operator bundle is release you can't change it.
> 
> If you want to update existing operators you would need to switch to File Based Catalog - #5031

@Allda Thank you for your suggestion. I will consider switching to FBC. Let me close this PR first.

**Pull request closed without merging**